screens/signup.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Devstore | devlopers mall</title>
+    <link rel="stylesheet" href="../styles/gobal.css" />
```

Comment:
and is this a typo in gobal? It should be global if that is not typo.
Use extension for spell check in VS code so you won't make typos.

Reply:
Thanks for pointing out this gobal soon going to global. xD